docs/prem-gateway/installation.md

```diff
@@ -0,0 +1,18 @@
+---
+id: installation
+title: Installation
+sidebar_label: Installation
+sidebar_position: 1
+---
+
+# Installation
+
+Prem-App and Prem-Daemon can run without Prem-Gateway.
+However, if you want to use Prem-App and Prem-Daemon in the Cloud environment, it is recommended to install Prem-Gateway cause of its features.<br/>
+It is enough to choose y/Y when prompted to install Prem-Gateway during the installation of Prem-App or Prem-Daemon. 
```

Comment:
Prem App instead of `Prem-App`. Same for `Prem-Daemon` and `Prem-Gateway`

Reply:
Done